openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/plugin.txt

```diff
@@ -39,25 +39,35 @@ VARIABLE reduced_log_retain_time 2592000
     LOG_RETAIN_TIME <%= log_retain_time %>
     REDUCED_LOG_RETAIN_TIME <%= reduced_log_retain_time %>
     TLM_LOG_CYCLE_TIME 300
+    # Allow the reducer microservice to take 50% of the cpu (default 30%)
+    REDUCER_MAX_CPU_UTILIZATION 50
 <% end %>
 
 <% if include_inst2 %>
   TARGET INST <%= inst2_target_name %>
     LOG_RETAIN_TIME <%= log_retain_time %>
     REDUCED_LOG_RETAIN_TIME <%= reduced_log_retain_time %>
     TLM_LOG_CYCLE_TIME 600
+    # Allow the reducer microservice to take 10% of the cpu (default 30%)
+    REDUCER_MAX_CPU_UTILIZATION 10
```

Comment:
This is really low for the heaviest target

Reply:
I'll change it ... but speaking of that. Can the reducer know it's falling behind and raise a notification?

Reply:
There are cases where that would be fine though.   Like if you dropped in a whole bunch of files to process at once and it just takes a long time to work through

Reply:
Probably worth a notification.